libs/libpam/Makefile

```diff
@@ -45,12 +45,16 @@ CONFIGURE_ARGS += \
 	--disable-audit \
 	--disable-cracklib \
 	--disable-db \
-	--disable-prelude \
+	--disable-debug \
+	--disable-doc \
+	--disable-econf \
 	--disable-lckpwdf \
 	--disable-nis \
+	--disable-prelude \
 	--disable-regenerate-docu \
 	--disable-rpath \
 	--disable-selinux \
+	--disable-Werror \
```

Comment:
This seems good in general. What kind of warnings are you trying to avoid?

Reply:
I think this is already default. I just made it explicit.

Format warnings have typically broken builds.